GpxKit/Gpx+Parser.swift

```diff
@@ -24,12 +32,22 @@ public extension Gpx {
 
 fileprivate extension XMLIndexer {
 
+    func metadata() -> Metadata? {
+        guard element != nil else { return nil }
+        let name = self["name"].element?.text
+        let description = self["description"].element?.text
+        let author = self["author"].element?.text
+        return Metadata(name: name, description: description, author: author)
+    }
+
     func point() throws -> Point {
         guard let element = element else { throw IndexingError.error }
         let latitude = try element.attributeValue(by: "lat")
         let longitude = try element.attributeValue(by: "lon")
-        return Point(coordinate: CLLocationCoordinate2D(latitude: latitude, longitude: longitude))
+        let time = self["time"].element?.text
+        return Point(coordinate: CLLocationCoordinate2D(latitude: latitude, longitude: longitude), time: time != nil ? TimeInterval(time!) : nil)
```

Comment:
You might be able to do `time: time as? TimeInterval)`

Reply:
This does't work. :( `TimeInterval` is a number and `time` is a `String`. We have to call the constructor to parse the string, I think.